package.json

```diff
@@ -47,29 +49,38 @@
     "gatsby-transformer-sharp": "^3.4.0",
     "hcaptcha": "^0.1.0",
     "isomorphic-dompurify": "^0.19.0",
-    "jsonwebtoken": "^8.5.1",
     "mailgun.js": "^6.0.1",
+    "https-browserify": "^1.0.0",
+    "isomorphic-dompurify": "^0.19.0",
```

Comment:
! duplicate of "isomorphic-dompurify"
```suggestion
```

Reply:
Done in my commit [faefe57](https://github.com/code-423n4/code423n4.com/pull/1186/commits/faefe57a9be65bc7a97aec57062757e5a9f410ab)